extras/conf/gluster-exporter.toml.sample

```diff
@@ -8,6 +8,15 @@ metrics-path = "/metrics"
 log-dir = "/var/log/gluster-exporter"
 log-file = "exporter.log"
 log-level = "info"
+cache-ttl-in-sec = 30
+# by default caching is enabled only for the below functions,
+# 'IsLeader', 'LocalPeerID', 'VolumeInfo'
+#
+# to enable caching, add the function-name to 'cache-enabled-funcs' list
+# other supported functions are,
+# 'EnableVolumeProfiling', 'HealInfo', 'Peers',
+# 'Snapshots', 'VolumeBrickStatus', 'VolumeProfileInfo'
+cache-enabled-funcs = []
```

Comment:
To make it uniform, add the enabled functions here instead of hard coding in the code.